### PR TITLE
Add possibility to disable bumping based on environment variable

### PR DESCRIPTION
**Pull request by pawbu**
This is for the testing phase of a data store.

### CODE REVIEW

**Review by haraldsvik — APPROVED** [review]
Looks good 💯

Remember to add the new environment variable to the services!

File: job_service/api/job_api.py [hunk 1]
```diff
@@ -35,14 +35,27 @@ def new_job(body: NewJobsRequest):
...
+                        "msg": "CREATED",
+                        "job_id": job.job_id,
+                    }
+                )
             target_db.update_target(job)
         except Exception as e:
             logger.exception(e)
-            response_list.append({"status": "FAILED", "msg": "FAILED"})
+            response_list.append(
+                {"status": "FAILED", "msg": f"FAILED: {str(e)}"}
```
Comment (DanielElisenberg) [inline job_service/api/job_api.py:57]: This is supposed to only catch unnamed unknown exceptions. Not ideal to log this information to the users 🙇🏻

File: job_service/api/job_api.py [hunk 2]
```diff
@@ -35,14 +35,27 @@ def new_job(body: NewJobsRequest):
...
-            job = job_db.new_job(job_request, user_info)
-            response_list.append(
-                {"status": "queued", "msg": "CREATED", "job_id": job.job_id}
-            )
+            if (
+                job_request.target == "DATASTORE"
+                and job_request.operation == "BUMP"
+                and environment.get("BUMP_ENABLED") is False
+            ):
+                raise Exception("Bumping the datastore is disabled")
```
Comment (DanielElisenberg) [inline job_service/api/job_api.py:43]: Raising a named exception here could solve the problem below 👍🏻

File: job_service/config/environment.py [hunk 3]
```diff
@@ -12,6 +12,11 @@ def _initialize_environment() -> dict:
         "JWT_AUTH": (
             False if os.environ.get("JWT_AUTH", "true") == "false" else True
         ),
+        "BUMP_ENABLED": (
```
Comment (DanielElisenberg) [inline job_service/config/environment.py:15]: :100:

File: job_service/api/job_api.py [hunk 4]
```diff
@@ -65,3 +78,18 @@ def update_job(body: UpdateJobRequest, job_id: str):
     if job.parameters.target == "DATASTORE" and job.status == "completed":
         target_db.update_bump_targets(job)
     return {"message": f"Updated job with jobId {job_id}"}
+
+
+def fail_bump_job_if_configured(job):
```
Comment (DanielElisenberg) [inline job_service/api/job_api.py:83]: is this function called?

File: job_service/api/job_api.py (hunk 1 above)
Comment (pawbu) [reply job_service/api/job_api.py:57]: I fix

File: job_service/api/job_api.py (hunk 4 above)
Comment (pawbu) [reply job_service/api/job_api.py:83]: Not any more 👍

**Review by DanielElisenberg — APPROVED** [review]
:100:

**Pull request merged**